docs/TuxAnimations.md

```diff
@@ -23,7 +23,72 @@ Tux is a modular framework and as such it is possible to require only the minima
 The only other step to implementing a Tux animation is to wrap the element you want animated with the desired animation component within the render method of your `React.createClass` function.
 
 ```
-  render: function() {
+  render: function () {
+    return (
+      <Fly>
+        <h1>Hello World</h1>
+      </Fly>
+    );
+  }
+```
+
+#####Specifying an Animation Key via the Id Property - type: STRING, ARRAY of STRINGs, or NUMBER - optional
+Tux Animations work by concatting the children of the animation into an array and wrapping that in a ReactTransitionGroup.  Because of that it is beneficial to provide the animation with an id property, which will tell it where to look in its children for a key to use.  If no id is provided the animations will still work, but you will see warnings from React regarding the lack of a key.  You can specify a string or array of strings for the id propery:
```

Comment:
Correct spelling of concatting to concatenating and correct spelling of property at end of line.